apps/site/src/app/schedule/components/EventRegular.tsx

```diff
@@ -32,25 +32,27 @@ interface EventMomentProps {
 }
 
 function EventTypeComponent({ eventType }: EventTypeProps) {
-	if (eventType === "Main") {
-		return (
-			<div className="inline-block px-4 py-1.5 font-semibold text-[#0D272D] bg-[#DFBA73] rounded-2xl sm:py-2">
-				{eventType}
-			</div>
-		);
-	} else if (eventType === "Workshop") {
-		return (
-			<div className="inline-block px-4 py-1.5 font-semibold text-[#0D272D] bg-[#94A9BD] rounded-2xl sm:py-2">
-				{eventType}
-			</div>
-		);
-	} else if (eventType === "Social") {
-		return (
-			<div className="inline-block px-4 py-1.5 font-semibold text-[#0D272D] bg-[#DFA9A9] rounded-2xl sm:py-2">
-				{eventType}
-			</div>
-		);
+	interface EventBackgroundColors {
+		Main: string;
+		Workshop: string;
+		Social: string;
 	}
+
+	const eventBackgroundColors: EventBackgroundColors = {
+		Main: "bg-[#DFBA73]",
+		Workshop: "bg-[#94A9BD]",
+		Social: "bg-[#DFA9A9]",
+	};
```

Comment:
Nitpick: It might be simpler to use [string enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) instead of an interface. Additionally, we generally declare constant declarations like these outside of the function.

Reply:
On second thought, string enums can't be used the way I thought they could be used, so please disregard my comment about using the enum, but still move it outside of the function.